<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialAssistantClass</class>
 <widget class="QWidget" name="SerialAssistantClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>580</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>SerialAssistant</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_8">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>Adobe Devanagari</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>串口配置</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="widget_7" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>端口：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="serialCom">
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>10</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QWidget" name="widget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>波特率：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="baudRate">
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="currentIndex">
               <number>2</number>
              </property>
              <item>
               <property name="text">
                <string>4800</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>9600</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>115200</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QWidget" name="widget_3" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>数据位：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="dataBit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <item>
               <property name="text">
                <string>8</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>7</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>6</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>5</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QWidget" name="widget_4" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>校验位：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="checkBit">
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <item>
               <property name="text">
                <string>none</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QWidget" name="widget_5" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>停止位：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="stopBit">
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <item>
               <property name="text">
                <string>1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1.5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_8" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPushButton" name="openSerial">
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>170</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 170, 0);
border-radius:10px;
color:rgb(0,0,0);</string>
           </property>
           <property name="text">
            <string>Open Serial</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cleanReceive">
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>170</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgb(255, 170, 0);
border-radius:10px;
color:rgb(0,0,0);</string>
           </property>
           <property name="text">
            <string>Clean Rece Area</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cleanSend">
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>170</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgb(255, 170, 0);
border-radius:10px;
color:rgb(0,0,0);</string>
           </property>
           <property name="text">
            <string>Clean Send Area</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="send">
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>170</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 170, 0);
border-radius:10px;
color:rgb(0,0,0);</string>
           </property>
           <property name="text">
            <string>Send Message</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_6" native="true">
     <property name="font">
      <font>
       <family>Adobe Devanagari</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="font">
         <font>
          <family>Adobe Devanagari</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="title">
         <string>接受区：</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QTextBrowser" name="receArea">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>发送区：</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPlainTextEdit" name="inputArea">
           <property name="font">
            <font>
             <family>Consolas</family>
            </font>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="SerialAssistant.qrc"/>
 </resources>
 <connections/>
</ui>
